cmd/detectExecuteScan.go

```diff
@@ -116,7 +117,31 @@ func runDetect(config detectExecuteScanOptions, utils detectUtils) error {
 	utils.SetDir(".")
 	utils.SetEnv(envs)
 
-	return utils.RunShell("/bin/bash", script)
+	err = utils.RunShell("/bin/bash", script)
+	if err == nil {
+		violations := struct {
+			PolicyViolations int      `json:"policyViolations"`
+			Reports          []string `json:"reports"`
+		}{
+			PolicyViolations: 0,
+			Reports:          []string{},
+		}
+
+		if exists, err := utils.FileExists("detect.risk.report.pdf"); err != nil && exists {
```

Comment:
The 'detect.risk.repor.pdf' parameter is used by detect script to generate the report. It is a boolean flag. The file name of the report itself will be different.
I will comment again after I find the right file name information.